.github/workflows/actions.yml

```diff
@@ -253,9 +256,21 @@ jobs:
         run: rm -rf $(brew --cache)
       - name: install dependencies
         run: |
-          brew install libsndfile portaudio ccache fftw
```

Comment:
`libsndfile` is already installed on the GHA runners, so this was redundant anyway